src/main/java/uk/nhs/digital/nhsconnect/nhais/model/mesh/MeshMessage.java

```diff
@@ -19,4 +19,5 @@
      * The timestamp (ISO format, UTC) when this message was sent - used for debugging and tracing
      */
     private String messageSentTimestamp;
+    private String meshMessageId;
```

Comment:
We don't need the message id for anything once it is published to the queue, but I think it is a good idea you've included it for traceability. We could even use this as the correlation id when distributed tracing is implemented 👍